### PR TITLE
DDF 1776 Fix css bugs introduced by ddf-1463.

### PR DESCRIPTION
**Pull request by shane-voisard**
@stustison @andrewkfiedler @jaymcnallie @jlcsmith

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/438" target="_blank">ReviewNinja

File: catalog/ui/search-ui/standard/src/main/webapp/css/style.css [hunk 1]
```diff
@@ -164,16 +164,16 @@ div.modal-body div.preferenceTabButtons {
     standardized selectors and properties and non-standard and prefixed selectors
     and properties."
 */
```
Comment (andrewkfiedler) [inline catalog/ui/search-ui/standard/src/main/webapp/css/style.css]: I would move all these input range stylings into a new .less file under ```ddf/catalog/ui/search-ui/standard/src/main/webapp/less/components/```.  You could call it ```range-input.less```.

File: catalog/ui/search-ui/standard/src/main/webapp/css/style.css [hunk 2]
```diff
@@ -103,20 +103,20 @@
     cursor: pointer;
 }
 
-#layerPickers table.scrollable {
+.layerPickers table.scrollable {
```
Comment (pklinef) [inline catalog/ui/search-ui/standard/src/main/webapp/css/style.css]: Previously style.css only contained CSS related to the header and footer and will someday be shared between projects in the future.  All of the layer picker CSS should be moved to searchPage.css or added to LESS.

**Comment by shane-voisard** [comment]
Refactored css according to comments by @pklinef and @andrewkfiedler .

**Comment by pklinef** [comment]
:+1:

**Pull request merged**